iceoryx_posh/include/iceoryx_posh/iceoryx_posh_types.hpp

```diff
@@ -100,14 +100,13 @@ namespace popo
 using WaitSetIsConditionSatisfiedCallback = cxx::ConstMethodCallback<bool>;
 }
 constexpr uint32_t MAX_NUMBER_OF_CONDITION_VARIABLES = 1024U;
-constexpr uint32_t MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE = 128U;
-constexpr uint32_t MAX_NUMBER_OF_ATTACHMENTS_PER_WAITSET = 128U;
-static_assert(MAX_NUMBER_OF_ATTACHMENTS_PER_WAITSET <= MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE,
-              "The WaitSet capacity is restricted by the maximum amount of notifiers per condition variable.");
-// Listener
-constexpr uint8_t MAX_NUMBER_OF_EVENTS_PER_LISTENER = 128U;
-static_assert(MAX_NUMBER_OF_EVENTS_PER_LISTENER <= MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE,
-              "The Listener capacity is restricted by the maximum amount of notifiers per condition variable.");
+
+constexpr uint32_t MAX_NUMBER_OF_NOTIFIERS_PER_CONDITION_VARIABLE =
```

Comment:
I tend to re-use the name because it is quite generic, a comment below should clarify that this is used by the Listener and Waitset API.

Reply:
Maybe we can shorten it a bit to `MAX_NUMBER_OF_NOTIFIERS`? The info that it belongs to a condition variable is not really important for the user.